public/pages/Findings/containers/Findings/Findings.tsx

```diff
@@ -151,30 +218,74 @@ class Findings extends Component<FindingsProps, FindingsState> {
 
   onRefresh = async () => {
     await this.getNotificationChannels();
-    await this.getPlugins();
-    await this.getFindings();
+    if (this.state.selectedTabId === FindingTabId.DetectionRules) {
```

Comment:
rename `selectedTabId` to `selectedTab`

Reply:
Same reasoning as above, tab is the entire object, we only store the id here